apps/meteor/app/livechat/server/hooks/verificationCheck.ts

```diff
@@ -28,12 +25,27 @@ callbacks.add(
 				break;
 			}
 			case RoomVerificationState.isListeningToOTP: {
+				const bot = await Users.findOneById('rocket.cat');
+				if (message.msg === 'Resend OTP') {
+					if (room.source.type === 'widget') {
+						const wrongOtpInstructionsMessage = {
+							msg: i18n.t('Visitor_Widget_Verification_Process_Resend_OTP'),
+							groupable: false,
+						};
+						await sendMessage(bot, wrongOtpInstructionsMessage, room);
+					} else if (room.source.type === 'app') {
```

Comment:
```suggestion
					} else {
```
I believe we'd need to make this an else since there can be other non-app related channels like SMS where we'd need to send these messages